ClarkeErrorGrid.py

```diff
@@ -66,77 +56,96 @@
 [5]     Guevara, E. and Gonzalez, F. J. (2010). Joint optical-electrical technique for
         noninvasive glucose monitoring, REVISTA MEXICANA DE FISICA, vol. 56,
         no. 5, pp. 430434.
-
-
 Made by:
 Trevor Tsue
 7/18/17
-
 Based on the Matlab Clarke Error Grid Analysis File Version 1.2 by:
 Edgar Guevara Codina
 codina@REMOVETHIScactus.iico.uaslp.mx
 March 29 2013
 '''
-
-
-
-import matplotlib.pyplot as plt
-
-
 #This function takes in the reference values and the prediction values as lists and returns a list with each index corresponding to the total number
 #of points within that zone (0=A, 1=B, 2=C, 3=D, 4=E) and the plot
-def clarke_error_grid(ref_values, pred_values, title_string):
+#Has optiional params for changing colors of lines, backgrounds, and datapoints. Can also change opacity of points and lables for the x and y axis.
+def clarke_error_grid(ref_values, pred_values, title_string, xlabel='Reference Concentration (mg/dL)', ylabel='Prediction Concentration (mg/dL)', 
+                      scatter_c='black', line_c='b', a_color = '#1874CD', b_color='#BFEFFF', c_color='#F8F8FF', regression_color = 'black',
```

Comment:
What does line_c do?
Recommend adding documentation for each of these variables in the `README.md`

Reply:
Also recommend ordering these variable in a sensible order